package.json

```diff
@@ -39,6 +41,12 @@
     "normalize.css": "^6.0.0",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
-    "react-scroll": "^1.5.2"
+    "react-scroll": "^1.5.2",
+    "react-pixnode-shuffle-images": "^1.1.1"
```

Comment:
Ahh, I understand that, and I agree to some extent. For hero unit, I prefer that the images are loaded immediately, because it’s the first thing that the user sees. This is similar to having a hero unit with a video background.

For images down below in the contents, I also agree that we should load images only when needed. 👍